src/boat_simulator/tests/unit/nodes/low_level_control/test_controller.py

```diff
@@ -0,0 +1,149 @@
+"""Tests classes and functions in boat_simulator/nodes/controller.py"""
+
+from math import atan2, cos, sin
+
+import numpy as np
+import pytest
+
+from boat_simulator.common.types import Scalar
+from boat_simulator.common.utils import bound_to_180
+from boat_simulator.nodes.low_level_control.controller import RudderController
+
+
+class TestRudderController:
```

Comment:
You can use pytest fixtures to define a rudder controller before each test so you don't have to redefine a new controller in every single test.